calculator-data/src/main/java/camp/nextstep/edu/calculator/data/MemoryDatabase.kt

```diff
@@ -0,0 +1,29 @@
+package camp.nextstep.edu.calculator.data
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+@Database(entities = [MemoryEntity::class], version = 1)
+abstract class MemoryDatabase : RoomDatabase() {
+    abstract fun resultDao(): MemoryDao
+
+    companion object {
+        private const val databaseName = "memory.db"
+        private var Instance: MemoryDatabase? = null
```

Comment:
```suggestion
        private var instance: MemoryDatabase? = null
```

코틀린에서 변수명의 첫글자는 소문자로 사용하는 것이 더욱 컨벤션에 맞습니다.

Reply:
네 맞습니다!
수정했습니다.